samples/Samples.HttpMessageHandler/Program.cs

```diff
@@ -75,6 +75,9 @@ public static void Main(string[] args)
 
         private static async Task SendHttpClientRequestAsync(bool tracingDisabled)
         {
+            // Insert a call to the Tracer.Instance to include an AssemblyRef to Datadog.Trace assembly in the final executable
```

Comment:
Why do we need this in this sample app and not the others?

Reply:
In this particular case I was trying to test a reference to just `Datadog.Trace` and not our `Datadog.Trace.ClrProfiler.Managed` assembly, to see if there would be any weird interactions with `Datadog.Trace.dll` already being in the bin folder. We do not need to have it there, but I thought it would be an interesting test.